CMakeLists.txt

```diff
@@ -245,5 +247,27 @@ message("CMAKE_CXX_FLAGS: ${CMAKE_CXX_FLAGS}")
 
 add_subdirectory(library)
 add_subdirectory(example)
-add_subdirectory(test)
+add_subdirectory(test EXCLUDE_FROM_ALL)
```

Comment:
Maybe we should rather have a cmake option like `-DWITH_TESTS=ON` instead of excluding by default all test targets from `all` target?

Reply:
I removed and fixed the issue elsewhere. The cause for this was that gmock was complaining on the warnings we set in our repos which has been overridden in the `googletest.cmake` file.